### ISSUE TITLE
duplicate directory entries in nfsrods mount

### ISSUE THREAD

**Issue by anderbubble** [irods/irods_client_nfsrods#40]
Our nfsrods mount uses lookupcache=none

```
[root@irods1 ~]# mount -t nfs4
localhost:/ on /mnt/nfsrods type nfs4 (rw,relatime,vers=4.1,rsize=1048576,wsize=1048576,namlen=255,hard,proto=tcp6,timeo=600,retrans=2,sec=sys,clientaddr=::1,lookupcache=none,local_lock=none,addr=::1)
```

But today I noticed that it was still returning duplicate directory entries.

```
[root@irods1 ~]# ls /mnt/nfsrods/home
frahm   holtat    jobl6604  monaghaa  pabi5658  public  shouse
frahm   joan5896  jobl6604  nfsrods   pabi5658  rods    shouse
holtat  joan5896  monaghaa  nfsrods   public    rods
```

re-mounting cleared this up; but it's troubling nonetheless.

```
[root@irods1 ~]# umount /mnt/nfsrods
[root@irods1 ~]# mount -v -t nfs4 -o sec=sys,port=2049,lookupcache=none localhost:/ /mnt/nfsrods
mount.nfs4: timeout set for Wed Sep 18 12:46:53 2019
mount.nfs4: trying text-based options 'sec=sys,port=2049,lookupcache=none,vers=4.1,addr=::1,clientaddr=::1'
[root@irods1 ~]# ls /mnt/nfsrods/home
frahm  holtat  joan5896  jobl6604  monaghaa  nfsrods  pabi5658  public  rods  shouse
```

**Comment by trel** [comment]
We saw this early on during development, but haven't seen it in a while...

Can you share the caching values in your `server.json` NFSRODS config?   Are they default?

**Comment by anderbubble** [comment]
```
[root@irods1 ~]# cat /etc/irods/nfsrods/server.json 
{
    "nfs_server": {
        "port": 2049,
        "irods_mount_point": "/curcZone",
        "user_information_refresh_time_in_milliseconds": 3600000,
        "file_information_refresh_time_in_milliseconds": 1000,
        "user_access_refresh_time_in_milliseconds": 1000
    },

    "irods_client": {
        "zone": "curcZone",
        "host": "irods1.rc.int.colorado.edu",
        "port": 1247,
        "default_resource": "demoResc",
        "ssl_negotiation_policy": "CS_NEG_REFUSE",
        "proxy_admin_account": {
            "username": "nfsrods",
            "password": "[redacted]"
        }
    }
}
```

**Comment by anderbubble** [comment]
Also notably this was the first access to the mountpoint in at least 18 hours.

**Comment by trel** [comment]
Settings look fine/default.     The 18 hours thing is interesting.    We'll continue to track it down.

**Comment by trel** [comment]
We expect the duplicate cookie fix (2174e8eae7be1ac59fcec24917a44f7f9fd40a17) will fix this issue as well.  Closing for now - can create a new issue if we see it again.
